### ISSUE TITLE
Feature Request: Add basemap toggle between streets tiles and satellite tiles

### ISSUE THREAD

**Issue by nlebovits** [CodeForPhilly/clean-and-green-philly#890]
**Is your feature request related to a problem? Please describe.**
When exploring individual properties on the map, I often like to see the satellite view in Google to understand what the property and its surroundings look like. Currently, I can only see the grey street tiles from Maptiler, so I have to open Google Maps separately to do this.

**Describe the solution you'd like**
Add a [style switcher control from Maptiler](https://docs.maptiler.com/sdk-js/examples/control-style-switcher/) to allow toggling between the "STREETS" and "HYBRID" basemaps. Make the "STREETS" tiles the default.

**Comment by AZBL** [comment]
I'd like to be assigned this please.

**Comment by AZBL** [comment]
Also @nlebovits  I was taking a look at this already. I see you mentioned being able to switch between STREETS and HYBRID and making streets the default. Currently, DATAVIZ is the default. Do you want to make it so you can switch between  dataviz and hybrid? 

https://docs.maptiler.com/sdk-js/examples/change-map-styles/

**Comment by nlebovits** [comment]
@AZBL yep, good catch. HYBRID and DATAVIZ is what I meant. Thanks for working on this!

**Comment by nlebovits** [comment]
This came up in our Nov. 14 workshop. I'd love to see someone take this on!

**Comment by DonovanAndrews300** [comment]
Hey I'd like to tackle this one!

**Comment by CodeWritingCow** [comment]
@DonovanAndrews300 assigned!

**Comment by nlebovits** [comment]
Closed
